buildSrc/src/main/groovy/org/elasticsearch/gradle/test/StandaloneTestPlugin.groovy

```diff
@@ -20,12 +20,10 @@
 package org.elasticsearch.gradle.test
```

Comment:
Can we go ahead and port this to Java too?

Reply:
unfortunately we can't at the moment. Whats blocking porting most groovy classes here to java is the AntTask class and classes inheriting this one. It makes use of groovys ant builder. I'm sure we can rewrite the `LicenseHeadersTask` task in a way to not rely on ant under the hood and then move most of the precommit stuff over to plain java. But not as part of this PR